### PR TITLE
Document known issues surrounding update-primary on PR secondaries.

### PR DESCRIPTION
**Pull request by ncabatoff**

### CODE REVIEW

File: website/content/docs/release-notes/1.14.0.mdx [hunk 1]
```diff
@@ -21,6 +21,8 @@ official guidance until the release has been completed.
 
 @include 'ui-pki-control-groups-known-issue.mdx'
 
+@include 'update-primary-known-issue.mdx'
```
Comment (mladlow) [inline website/content/docs/release-notes/1.14.0.mdx:24]: Could you please add to 1.13 release notes & upgrade guide as well?

File: website/content/partials/update-primary-known-issue.mdx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+If update-primary is done on a PR secondary with shared mounts containing
+[local data](/vault/docs/enterprise/replication#replicated-data) (e.g. pki certs,
+approle secretids), the merkle tree on the PR secondary may get corrupted due to
+a timing race.
+
+When this happens, the PR secondary still contains all the stored data, e.g. listing
+local certs from PKI mounts will return the correct results.  However, because the
+merkle tree has been corrupted, a downstream DR secondary will not receive the local
+data, and will delete it if it already had it.  If the DR secondary is promoted before
```
Comment (mladlow) [inline website/content/partials/update-primary-known-issue.mdx:28]: Maybe?
```suggestion
data, and will delete it if it already had it.  If the PR secondary's DR secondary is promoted before
```

**Review by jasonodonnell — APPROVED** [review]

File: website/content/docs/release-notes/1.14.0.mdx (hunk 1 above)
Comment (ncabatoff) [reply website/content/docs/release-notes/1.14.0.mdx:24]: Yeah, I was wondering how far back we should go.  Just 1.13 then?

File: website/content/partials/update-primary-known-issue.mdx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+If update-primary is done on a PR secondary with shared mounts containing
+[local data](/vault/docs/enterprise/replication#replicated-data) (e.g. pki certs,
+approle secretids), the merkle tree on the PR secondary may get corrupted due to
+a timing race.
+
+When this happens, the PR secondary still contains all the stored data, e.g. listing
+local certs from PKI mounts will return the correct results.  However, because the
+merkle tree has been corrupted, a downstream DR secondary will not receive the local
+data, and will delete it if it already had it.  If the PR secondary's DR secondary is promoted before
+the PR secondary is repaired, the newly promoted PR secondary will not contain the loca
```
Comment (maxb) [inline website/content/partials/update-primary-known-issue.mdx:29]:
```suggestion
the PR secondary is repaired, the newly promoted PR secondary will not contain the local
```

**Pull request merged**